platform/qemu.go

```diff
@@ -318,12 +327,17 @@ func CreateQEMUCommand(board, uuid, biosImage, consolePath, confPath, diskImageP
 		plog.Debugf("disabling auto-read-only for QEMU drives")
 	}
 
-	allDisks := append([]Disk{
-		{
-			BackingFile: diskImagePath,
-			DeviceOpts:  primaryDiskOptions,
-		},
-	}, options.AdditionalDisks...)
+	primaryDisk := Disk{
+		BackingFile: diskImagePath,
+		DeviceOpts:  primaryDiskOptions,
+		ConfPath:    "",
+	}
+
+	if board == "s390x-usr" {
```

Comment:
@arithx I know the `*-usr` convention is what we had in ContainerLinux. Are we sticking to that also for the new *COS images?

Reply:
Good question. I see [above in the code it's currently still in use](https://github.com/coreos/mantle/pull/1050/files#diff-da5387f5087a8b96b97f913fdefab572R241).

Reply:
@lucab There isn't anything necessarily restricting us to that convention, however as it's just a user provided parameter to inform `kola` that it's targeting that board architecture I don't think that there's necessarily any harm in it.

Reply:
I'd rather not perpetuate the `board` and `-usr` naming if we don't have to.

Reply:
Should we only use `s390x` here?

Reply:
I talked with @bgilbert yesterday about this. I'm going to investigate some other approaches for *COS distributions outside of this issue and we won't block on it.